config/packages/framework.yaml

```diff
@@ -24,5 +24,12 @@ framework:
         app: cache.adapter.redis
         default_redis_provider: '%env(REDIS_CACHE_DSN)%'
 
+        pools:
+            # Creates a "no.hits.cache" service
+            no.hits.cache:
+                adapter: cache.adapter.redis
+                # Default expire set to one day
+                default_lifetime: 172800
```

Comment:
Could we make this configurable with a parameter from `.env`?

Reply:
Done